### DISCUSSION TITLE
Ayuda con la practica JS pokedex

### DISCUSSION THREAD

**Discussion by charlyguz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#520] [Q&A] [answered]
Alguien me puede ayudar con este error, tengo este error desde hace unos dias, ya quiero entregar la practica pero a la hora de desplegar la pagina en github pages aparece muy diferente a como yo lo veo,  asi se ve cuando la despliego en github pages
![image](https://user-images.githubusercontent.com/83683712/160076099-ee8be3c9-b5b8-448a-a99a-5b9a3171cc97.png)

y asi deberia de verse o asi la veo yo cuando la edito localmente:
![CAPTURA pokedex](https://user-images.githubusercontent.com/83683712/160076308-303e71d9-a9fc-4489-8dd8-e5de15446d81.jpg)

**Comment by marco-cortes** [comment]
Lo más probable es que sea que no encuentra la imagen de fondo, para que arregles eso, si tu ruta es por ejemplo:
ruta/imagen_fondo.jpg
Cámbialo a
./ruta/imagen_fondo.jpg
Aunque ojo, esta ruta puede cambiar porque cuando usas el .
La ruta se toma desde el archivo donde estas importando la imagen.

**Comment by ismagician** [comment, marked as answer]
Hola, checando el código de tu repositorio la ruta de la imagen en el css esta mal tienes
 background-image: url("../JS/assets/images/POKEMON.jpg"); pero  debe ser  background-image: url("./assets/images/POKEMON.jpg"); 
 También no está la imagen de fondo que quieres poner en el repositorio, encontré una diferente te muestro como queda con el código modificado:
 
 
![imagen](https://user-images.githubusercontent.com/92187317/160168565-a6400392-bced-43fb-87af-d797373e8fe5.png)

 
![imagen](https://user-images.githubusercontent.com/92187317/160167056-b8cc449a-8c00-4a4f-b06e-1cdb1a34b1ee.png)

**Reply by charlyguz** [reply]
hola, hice los cambios que me mostraste pero sigue sin aparecer el fondo :(

**Reply by ismagician** [reply]
Ya carga la imagen, como dice @damipere  borra tu cache o prueba con otro navegador
![imagen](https://user-images.githubusercontent.com/92187317/160249552-83887ab7-264b-45a6-8ef9-35f9ebccd8d8.png)

**Comment by damipere** [comment]
borra tu cache si carga la imagen
![imagen](https://user-images.githubusercontent.com/99141604/160218976-78816ce8-d3a3-4867-81a3-c1a9d0744d83.png)
